### PR TITLE
Typo fix :)

### PR DESCRIPTION
**Pull request by egonw**

### CODE REVIEW

File: ontology/lipro.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ build:
   source_url: http://www.lipidprofiles.com/LipidOntology
 contact:
   email: bakerc@unb.ca
-  label: Christipher Baker
```
Comment (cthoyt) [inline ontology/lipro.md:10]: @egonw do you know the ORCID and github handles for this person?

Comment (egonw) [reply ontology/lipro.md:10]: Should be him:

* https://orcid.org/0000-0003-4004-6479
* https://scholia.toolforge.org/author/Q59389381

Comment (cthoyt) [reply ontology/lipro.md:10]: can you add this also under `label:` as `orcid: 0000-0003-4004-6479` then we can accept this PR!

**Review by cthoyt — APPROVED** [review]
@matentzn this one is also ready

**Review by matentzn — APPROVED** [review]

**Pull request merged**